amqpdispatcher/dispatcher_common.py

```diff
@@ -315,7 +315,7 @@ def setup(logger_name, connector, connect_to_hosts):
         )
 
         consume_channel.basic_consume(
-            consumer_callback=pool.handle,
+            pool.handle,
```

Comment:
Is this tested against python2? Not all your codebases are python3.

Reply:
we'll get there we'll get there